test/alternator/test_table.py

```diff
@@ -553,3 +554,55 @@ def creates():
                     time.sleep(1)
                     continue
                 raise
+
+# Test that DeleteTable returns correct TableDescription (issue #11472)
+def test_delete_table_description(dynamodb):
+    table_name = 'alternator_test'
```

Comment:
Please use a unique table name (e.g., "unique_table_name()") instead of a fixed string. These fixed strings can become a problem if you want to repeat the same test multiple times, for example, and didn't wait for the table to be deleted completely before starting the next iteration.

Reply:
Done